src/server/index.js

```diff
@@ -33,6 +34,9 @@ function setupFetch(server) {
 }
 
 async function main() {
+  // Generate manifest.json respecting theme
+  generateManifestJson()
```

Comment:
Instead of writing this to disk, can we just add one more route under `/manifest.json` and return that JSON?

Reply:
I think that would be the appropriate approach when the manifest is fully dynamic (i.e. the themes are stored against a user, for example).
In this case, since the theme cannot change after init, it would be extra server work to generated at request time, even if nominal.